base/applications/osk/lang/ar-DZ.rc

```diff
@@ -61,11 +61,23 @@ END
 
 STRINGTABLE
 BEGIN
-    STRING_OSK "On-Screen Keyboard"
-    STRING_AUTHORS "Copyright Denis Robert"
+    IDS_OSK "On-Screen Keyboard"
+    IDS_AUTHORS "Copyright Denis Robert"
     IDS_NUMLOCK "Num"
     IDS_CAPSLOCK "Caps"
     IDS_SCROLLLOCK "Scroll"
+    /* Make sure the text fits onto the buttons */
+    IDS_ESCAPE "Esc"
+    IDS_PRN "Prn"
+    IDS_STOP "Stop"
+    IDS_ATTN "Attn"
+    IDS_INSERT "ins"
+    IDS_NUMLOCKKEY "nlk"
+    IDS_DELETE "del"
+    IDS_END "end"
+    IDS_CTRL "ctrl"
+    IDS_LEFTALT "alt"
+    IDS_RIGHTALT "alt"
```

Comment:
hmm... Why adding translations for these? Aren't they already "translated" (and localized) from the keyboard layouts?

Reply:
`ToUnicode` does not work with these keys

Reply:
Then how did you manage to get the screenshots (from yesterday) before you added this commit today?

Reply:
The key texts are in the keyboard structs for defaults in case if ToUnicode fails (for characters) and for system keys, they are just directly being taken (if translate field in KEY struct is FALSE).
(The file itself was initially was just a scripted conversion from the dialog templates)